gc/base/MarkingScheme.hpp

```diff
@@ -289,9 +289,13 @@ class MM_MarkingScheme : public MM_BaseVirtual
 	 */
 
 	MMINLINE void fixupForwardedSlot(GC_SlotObject *slotObject) {
-		omrobjectptr_t slot = slotObject->readReferenceFromSlot();
-		fixupForwardedSlot(&slot);
-		slotObject->writeReferenceToSlot(slot);
+#if defined(OMR_GC_CONCURRENT_SCAVENGER)
```

Comment:
to make platforms that don't have OMR_GC_CONCURRENT_SCAVENGER enabled (standalone OMR is one) move the ifdef to inner method.
The outer method does not need it (isConcurrentScavengerEnabled and isScavengerBackOutFlagRaised are always defined)
The inner method needs it since some FH are only defined with  CS ifdef.